client/src/pages/Login.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from "react";
+
+/**
+ * Login page for parents
+ */
+class Login extends Component {
+  constructor() {
+    super();
+    this.state = {
+      email: "",
+      password: "",
+      errors: {}
+    };
+  }
+
+  onChange = e => {
+    this.setState({ [e.target.id]: e.target.value });
+  };
+  
+  onSubmit = e => {
+    // prevent page from reloading
+    e.preventDefault();
+    const householdData = {
+      email: this.state.email,
+      password: this.state.password
+    };
+    console.log(householdData);
+    // TODO: call the api route for login
+  };
+
+  render() {
+    const { errors } = this.state;
+    return (
+      <div className="container">
+        <div style={{ marginTop: "4rem" }} className="row">
+          <div className="col s8 offset-s2">
+            <div className="col s12" style={{ paddingLeft: "11.250px" }}>
+              <h4>
+                <b>Login to Bright Futures Academy</b>
+              </h4>
+              <p className="grey-text text-darken-1">
+                Don't have an account? <a href={"mailto:brightFuturesAcademyDev@gmail.com"}>Contact the Administrator</a>
```

Comment:
Seems to be a github parsing issue here, ignore this. It renders properly on the front end